tools/crm_verify.c

```diff
@@ -70,8 +71,14 @@ static GOptionEntry addl_entries[] = {
     { NULL }
 };
 
+static pcmk__supported_format_t formats[] = {
+    PCMK__SUPPORTED_FORMAT_TEXT,
+    PCMK__SUPPORTED_FORMAT_XML,
```

Comment:
Should we support NONE as a general rule?

Reply:
Yeah probably.  There's no reason not to, and it could potentially be useful for scripting or something (get rid of all output and just have a return code).